gossip/src/crds.rs

```diff
@@ -169,6 +171,7 @@ impl Default for Crds {
             votes: BTreeMap::default(),
             epoch_slots: BTreeMap::default(),
             duplicate_shreds: BTreeMap::default(),
+            restart_last_voted_fork_slots: BTreeMap::default(),
```

Comment:
If these values are only going to matter during restart, then we don't need the overhead of this index.
You can just use this function:
https://github.com/solana-labs/solana/blob/6db57f81d/gossip/src/crds.rs#L380-L390
and filter on `RestartLastVotedForkSlots`.

Reply:
That's fair, removed now.